src/site/content/en/example-collection/index.njk

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
Let's move this file to https://github.com/GoogleChrome/web.dev/blob/main/src/site/content/en/handbook/content-types/ folder